app/decorators/person_decorator.rb

```diff
@@ -134,6 +134,11 @@ def upcoming_events
     @upcoming_events ||= EventDecorator.decorate_collection(event_queries.upcoming_events)
   end
 
+  def upcoming_participations
+    @upcoming_participations ||=
+      Event::ParticipationDecorator.decorate_collection(event_queries.upcoming_participations)
+  end
```

Comment:
Aus meiner Sicht macht es mehr Sinn, aus den `upcoming_participations` die Events herauszuholen aus den `upcoming_events` wieder die Participations. Deshalb ist das hier dazu gekommen.

Reply:
Ja finde ich gut. Vielleicht muss man noch in den Wagons kurz durchschauen ob es irgendwo eine Anpassung an `upcoming_events` gab die man jetzt für den Wagon mitziehen muss (wahrscheinlich nicht aber just to be sure)

Reply:
Es gibt bei der Jubla noch das: https://github.com/hitobito/hitobito_jubla/blob/455e3f4fafaef94c98360bfda59630b447ad472b/app/domain/jubla/person/event_queries.rb#L26